Sources/StreamVideo/OpenApi/generated/Models/EgressRTMPResponse.swift

```diff
@@ -1,37 +1,41 @@
 //
-// EgressRTMPResponse.swift
-//
-// Generated by openapi-generator
-// https://openapi-generator.tech
+// Copyright © 2024 Stream.io Inc. All rights reserved.
 //
 
 import Foundation
-
-
-public struct EgressRTMPResponse: Codable, JSONEncodable, Hashable {
+    
+public final class EgressRTMPResponse: @unchecked Sendable, Codable, JSONEncodable, Hashable {
```

Comment:
I don't see this one used anywhere?

Reply:
This is a relatively new one. Used in `EgressResponse` -> `CallResponse`.